Cargo.toml

```diff
@@ -4,3 +4,10 @@ version = "0.1.0"
 authors = ["The Chromium OS Authors"]
 license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
+
+[features]
+# This feature is used to "disable" long running tests. This feature should be
+# used to exclude long running tests (for example, tests that need large memory
```

Comment:
disable -> enable? exclude -> include?
Also this could probably be briefly documented somewhere in README as well.

Reply:
The README should be for user facing documentation. We can add a contributing document, but I find it to be too much for just this feature because you should only use it if you're writing long running tests.

If you are writing long running tests, than you most likely will see the existing one. I think the documentation outside of the code in this context will only rot, as we'll probably forget about it.

Reply:
By default when you are running `cargo test`, it will not run the tests that are under a feature, so that's why I used "exclude" and "disable". As the effect of using this feature is to disable them by default, and you can only enable them by explicitly writing `--features long_running_test`.

Reply:
Hmmm, when reading `This feature should be used to exclude long running tests`, I understand that if I pass that feature, then the tests will be excluded. Maybe the word `used` is not the best in this context.

I was mentioning the README because it says: `Does the crate have any default/optional features? What is each feature doing?` :satisfied:

Reply:
> Hmmm, when reading `This feature should be used to exclude long running tests`, I understand that if I pass that feature, then the tests will be excluded. Maybe the word `used` is not the best in this context.
> 

I reworded this. Can you please take another look?

> I was mentioning the README because it says: `Does the crate have any default/optional features? What is each feature doing?`

I explicitly called out that users should not use this in production the README as we discussed offline.